src/rules/no-unsafe-values.js

```diff
@@ -13,6 +19,11 @@ export default {
 
 		messages: {
 			unsafeNumber: "Number outside safe range found.",
+			unsafeInteger: "Integer outside safe range found.",
+			unsafeZero:
+				"This number will evaluate to zero, which is unintended.",
+			subnormal:
+				"Subnormal numbers (see https://en.wikipedia.org/wiki/Subnormal_number) are outside the safe range: '{{ value }}'.",
```

Comment:
We don't put URLs in messages as that makes the message more difficult to read.
```suggestion
				"Unexpected subnormal number '{{ value }}' found. Subnormal numbers are outside the safe range.",
```